### PR TITLE
Add one test case.

### PR DESCRIPTION
**Pull request by RdecKa**
I've added one simple test case. Can you tell if it looks like it should look like? :)
I've never used C++ before so I don't really know which data structures to use ...

### CODE REVIEW

File: test/testmain.cpp [hunk 1]
```diff
@@ -102,3 +102,26 @@ TEST_CASE("flex parser") {
...
+
+  Lexer* lexer;
+  lexer = new FLexer(sf, sym);
+  Token token;
+  for (auto test_tuple : expected_output) {
+    lexer->get_next(token);
+    REQUIRE(token.lexeme == std::get<0>(test_tuple));
+    REQUIRE(token.line == std::get<1>(test_tuple));
+    REQUIRE(token.type == std::get<2>(test_tuple));
+  }
```
Comment (csssaz) [inline test/testmain.cpp:127]: Can you add the line "delete lexer;" at the end in this and the other test I did?

I forgot adding it back then and just realised here 😅 . Nothing should happen in this case but it's a memory leak anyways.

**Review by csssaz — CHANGES_REQUESTED** [review]
it looks good! just take a look at my comment please (was my bad sorryy). 

you don't need to use any structure in particular specifically in this case, you could in fact just add a bunch of REQUIREs but that's more difficult in my opinion :b. tuples are super neat and modern c++ish!

File: test/testmain.cpp (hunk 1 above)
Comment (RdecKa) [reply test/testmain.cpp:127]: Fine now?

**Review by csssaz — APPROVED** [review]

File: test/testmain.cpp (hunk 1 above)
Comment (csssaz) [reply test/testmain.cpp:127]: 🚀

**Pull request merged**